core/emu.sh

```diff
@@ -1,36 +1,43 @@
 #!/bin/bash
 
-source emu.config 2>/dev/null
```

Comment:
Please return this back. We want to keep the possibility of having options in `emu.config`, no?

Reply:
Ah, I see it is included in a different place. Sorry.